client/rest-high-level/src/test/java/org/elasticsearch/client/eql/EqlSearchRequestTests.java

```diff
@@ -35,30 +35,33 @@
 
     @Override
     protected EqlSearchRequest createClientTestInstance() {
-        EqlSearchRequest EqlSearchRequest = new EqlSearchRequest("testindex", randomAlphaOfLength(40));
+        EqlSearchRequest eqlSearchRequest = new EqlSearchRequest("testindex", randomAlphaOfLength(40));
         if (randomBoolean()) {
-            EqlSearchRequest.fetchSize(randomIntBetween(1, Integer.MAX_VALUE));
+            eqlSearchRequest.fetchSize(randomIntBetween(1, Integer.MAX_VALUE));
         }
         if (randomBoolean()) {
-            EqlSearchRequest.eventCategoryField(randomAlphaOfLength(10));
+            eqlSearchRequest.size(randomInt(Integer.MAX_VALUE));
```

Comment:
Shouldn't this be `randomIntBetween(0, Integer.MAX_VALUE)`? or maybe to cover better the 0 scenario - a `randomBoolean()` to choose between `size` 0 and `size` between 1 and MAX.

Reply:
`randomInt` is `randomIntBetween(0, Integer.MAX_VALUE)`